src/components/helpers.tsx

```diff
@@ -18,7 +18,11 @@ export const block = (margin: number = 0) => {
 
 const sizes = theme.flexboxgrid.breakpoints
 
-export const media: any = Object.keys(sizes).reduce((accumulator, label) => {
+type Media = {
+  [S in keyof typeof sizes]: typeof css
+}
```

Comment:
So, what’s up with this, right?

* First we get a `union type` that consist of the keys of [the `sizes` object](https://github.com/artsy/reaction-force/blob/afa0a8d3811a10cea11f2c0affe2bd13436f1061/src/assets/theme.ts#L12-L15) with `keyof typeof sizes`, which results in something like: `type SizeKeys = "xs" | "sm" | "md" | "lg"`.
* Then we create a `mapped type` by iterating over the union elements with the `S in ...` syntax to produce the keys of the `media` object and we type the values to be the same of the `css` function, which is the function we wrap internally and pass arguments verbatim.

The result is:

```ts
type Media = {
    xs: ThemedCssFunction<any>
    sm: ThemedCssFunction<any>
    md: ThemedCssFunction<any>
    lg: ThemedCssFunction<any>
}
```